lib/models/theme.dart

```diff
@@ -22,70 +16,119 @@ import 'package:flutter/material.dart';
-    onTertiary: _ThcColors.darkMagenta,
-    error: _ThcColors.red,
-    onError: _ThcColors.white,
-    background: _ThcColors.darkBlue,
-    onBackground: _ThcColors.paleAzure,
-    surface: _ThcColors.dullBlue,
-    onSurface: _ThcColors.paleAzure,
-  ),
+const _darkColors = ColorScheme(
+  brightness: Brightness.dark,
+  primary: ThcColors.green,
+  onPrimary: Colors.white,
+  primaryContainer: ThcColors.darkGreen,
+  onPrimaryContainer: Colors.white,
+  secondary: ThcColors.teal,
+  onSecondary: Colors.white,
+  tertiary: ThcColors.tan,
+  onTertiary: ThcColors.darkMagenta,
+  error: Colors.red,
+  onError: Colors.white,
+  background: ThcColors.darkBlue,
+  onBackground: ThcColors.paleAzure,
+  surface: ThcColors.dullBlue,
+  onSurface: ThcColors.paleAzure,
 );
 
+const _iconTheme = IconThemeData(size: 32);
+const _labelTextStyle = TextStyle(fontWeight: FontWeight.w600, fontSize: 12);
+const _buttonStyle = ButtonStyle(shape: MaterialStatePropertyAll(BeveledRectangleBorder()));
+
+AppBarTheme _appBarTheme(bool isDark) => AppBarTheme(
+      backgroundColor: ThcColors.darkBlue,
+      foregroundColor: isDark ? ThcColors.paleAzure : Colors.white,
+    );
+
+/// [MaterialStateProperty] is pretty neat: you can have different styles
+/// based on whatever's going on with the widget.
+///
+/// For example, if a Director clicks on the "stream" button
+/// and then keeps their mouse hovering over it, then
+/// the button's set of Material states would look like this:
+///
+/// ```dart
+/// states = {MaterialState.hovered, MaterialState.selected};
+/// ```
+///
+/// Since we're using this [_tealWhenSelected] function, the color will be [ThcColors.teal].
+MaterialStateProperty<T> _tealWhenSelected<T>(T Function({Color color}) copyWith) =>
+    MaterialStateProperty.resolveWith((states) => states.contains(MaterialState.selected)
+        ? copyWith(color: ThcColors.teal)
+        : copyWith(color: Colors.white));
+
+NavigationBarThemeData _navigationBarTheme(bool isDark) => NavigationBarThemeData(
+      backgroundColor: isDark ? Colors.transparent : ThcColors.darkBlue,
+      indicatorColor: Colors.transparent,
+      iconTheme: _tealWhenSelected(_iconTheme.copyWith),
+      labelTextStyle: _tealWhenSelected(_labelTextStyle.copyWith),
+      labelBehavior: NavigationDestinationLabelBehavior.onlyShowSelected,
+    );
+
+ThemeData _themeScheme(ColorScheme scheme) {
+  final bool isDark = scheme.brightness == Brightness.dark;
+  return ThemeData(
+    colorScheme: scheme,
+    materialTapTargetSize: MaterialTapTargetSize.padded,
+    filledButtonTheme: const FilledButtonThemeData(style: _buttonStyle),
+    elevatedButtonTheme: const ElevatedButtonThemeData(style: _buttonStyle),
+    appBarTheme: _appBarTheme(isDark),
+    navigationBarTheme: _navigationBarTheme(isDark),
+  );
+}
+
+/// {@macro models.theme.colorScheme}
+final lightTheme = _themeScheme(_brightColors);
+
+/// {@macro models.theme.colorScheme}
+final darkTheme = _themeScheme(_darkColors);
```

Comment:
I expanded our `ThemeData`; now it's a bit more than just the color scheme.

The `_themeScheme()` function is my best effort to avoid lots of redundant values.